index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Setup and mobile version</title>
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <h2 class="name">Mucyo</h2>
+
+        <i class="fa fa-bars"></i>
+      </nav>
+    </header>
+    <section class="headline-page">
+      <div class="introduction">
+        <h1 class="intro">Hey There.<br />I'm Mucyo</h1>
+        <h2 class="headline-page-h2">I'am a Software Developer</h2>
+
+        <p class="text-1">
+          I can help you build a product, feature or website <br />Look through
+          some of my work and experience!<br />
+          if you like what you see and have a project <br />
+          you need coded, don't hestiate to contacta me.
+        </p>
+      </div>
+
+      <ul class="social-icons">
+        <li class="fa-size">
+          <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+        </li>
+        <li class="fa-size">
+          <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+        </li>
+        <li class="fa-size">
+          <a href="#"><i class="fa fa-medium" aria-hidden="true"></i></a>
+        </li>
+        <li class="fa-size">
+          <a href="#"><i class="fa fa-github" aria-hidden="true"></i></a>
+        </li>
+        <li class="fa-size">
+          <a href="#"><i class="fa fa-angellist" aria-hidden="true"></i></a>
+        </li>
+      </ul>
+    </section>
+    <div class="middle_header">
+      <h1>My Recent Works</h1>
+    </div>
+    <hr class="new5" />
+    <div class="middle_container">
+      <img src="images/Img.png" alt="image" class="middle-image" />
+      <div class="main_block">
+        <div class="post">Multi-Post Stories</div>
+        <div class="text_post">
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+        </div>
+        <div class="buttonsn">
+          <ul>
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
```

Comment:
- [ ] your project looks a bit different from the original design.
![image](https://user-images.githubusercontent.com/43698511/129424556-874dc16f-683b-405e-88de-f228d6f75deb.png)
Kindly fix the style of this list. You should respect the margins and font-family used in the original design.